### ISSUE TITLE
Lack of keyboard accessibility

### ISSUE THREAD

**Issue by RinkAttendant6** [rstacruz/details-polyfill#10]
It appears that the polyfill doesn't provide any keyboard accessibility to toggle the details element. I tried adding `tabindex=0` to the `<summary>` element, which makes it focusable but not toggleable.

Both Chrome and Firefox allow the user to focus on the `<summary>` element and toggle the details using the <kbd>Space</kbd> or <kbd>Enter</kbd> keys.

It is probably not too difficult to implement, if I do get something working I will send over a pull request
